cmake/KokkosFFT_tpls.cmake

```diff
@@ -2,34 +2,68 @@
 #
 # SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
 
-function(get_tpls_list tpls_list)
+function(get_tpls_list tpls_list backend_list)
```

Comment:
Please improve the documentation of the function to tell what it does, and what the output variables are about. There are no official documentation formats for CMake functions, but [this unofficial one](http://doc.aldebaran.com/1-14/qibuild/contrib/cmake/coding_guide.html) was pretty useful.

Reply:
Not sure what is the right way. But I added a cstyle docstring